scripts/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { Stats, stat, readFile } from 'fs';
+import * as glob from 'glob';
+import { ncp } from 'ncp';
+import * as path from 'path';
+import * as pify from 'pify';
+import { ChildProcess, ForkOptions, fork, SpawnOptions, spawn } from 'child_process';
+import * as once from 'once';
+
+export const BUILD_DIRECTORY = 'build';
+
+export const globP: (pattern: string) => Promise<string[]> = pify(glob);
+export const ncpP: (src: string, dest: string) => Promise<void> = pify(ncp);
+export const statP: (path: string) => Promise<Stats> = pify(stat);
+export const readFileP: (path: string, encoding?: string) => Promise<Buffer|string> = pify(readFile);
+
+export async function existsP(path: string): Promise<boolean> {
+  let exists = true;
```

Comment:
Not sure which is cleaner but this could also be something like:

```javascript
return new Promise(res => { fs.access(path, err => { res(!err); }) });
```

Reply:
Actually, I can do this which might be cleaner than either:

```js
return statP(path).then(
  () => Promise.resolve(true),
  () => Promise.resolve(false)
);
```